### PR TITLE
Fix confusing issue title with NULL crash state.

### PR DESCRIPTION
**Pull request by inferno-chromium**
Fixes
https://github.com/google/clusterfuzz/issues/579

### CODE REVIEW

File: src/appengine/handlers/cron/cleanup.py [hunk 1]
```diff
@@ -744,8 +744,8 @@ def _get_severity_from_labels(security_severity_label, labels):
   return data_types.SecuritySeverity.MISSING
 
 
-def _update_issue_severity_labels(policy, testcase, issue):
-  """Update severity labels on issue."""
+def _get_security_severity_issue_comment(policy, testcase, issue):
```
Comment (oliverchang) [inline src/appengine/handlers/cron/cleanup.py]: nit: maybe something like _update_issue_severity_and_get_comment() to make it clear that it's updating the severity on the issue too?

Comment (inferno-chromium) [reply src/appengine/handlers/cron/cleanup.py]: My bad, i read the function wrong and then did this rename. Fixed it now, thanks for catching.

**Pull request merged**